documentation/asciidoc/computers/raspberry-pi/bootmodes.adoc

```diff
@@ -4,7 +4,7 @@ The Raspberry Pi has a number of different stages of booting. This document expl
 
 === Special `bootcode.bin`-only boot mode
 
-USB host and Ethernet boot can be performed by BCM2837-based Raspberry Pis - that is, Raspberry Pi 2B version 1.2, Raspberry Pi 3B, and Raspberry Pi 3B+ (Raspberry Pi 3A+ cannot net boot since it does not have a built-in Ethernet interface). In addition, all Raspberry Pi models *except Raspberry Pi 4B* can use a new `bootcode.bin`-only method to enable USB host boot.
+USB host and Ethernet boot can be performed by BCM2837-based Raspberry Pis - that is, Raspberry Pi 2B version 1.2, Raspberry Pi 3B, and Raspberry Pi 3B+ (Raspberry Pi 3A+ cannot net boot since it does not have a built-in Ethernet interface). In addition, all Raspberry Pi models *prior to Raspberry Pi 4* can use a new `bootcode.bin`-only method to enable USB host boot.
```

Comment:
While you're in there, it might be worth taking out the word `new` - it's a few years old now 😉